src/libraries/System.Data.Common/src/System.Data.Common.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <AssemblyName>System.Data.Common</AssemblyName>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <NoWarn>$(NoWarn);CS1573</NoWarn>
+    <NoWarn>$(NoWarn);CS1573;CA2249</NoWarn>
```

Comment:
Why was this rule disabled here? (Questioning it because this project only targets netcoreapp)

Reply:
I honestly had the same questions. The CI build failed here without suppressing CA2249 even though the target says netcoreapp. I just assumed out build infra was also building this for another config somehow. Actually let's test it again now